etc/webpack/dev_server.js

```diff
@@ -27,7 +27,9 @@
 const errorOverlayMiddleware = require('react-dev-utils/errorOverlayMiddleware');
 const evalSourceMapMiddleware = require('react-dev-utils/evalSourceMapMiddleware');
 const noopServiceWorkerMiddleware = require('react-dev-utils/noopServiceWorkerMiddleware');
+const path = require('path');
```

Comment:
Neither this import nor the `./paths` import below appear to be used in this file. Are these necessary? Was there a reason why they were added?

Reply:
@duaneatat This comment still applies correct?